src/polyswarmclient/utils.py

```diff
@@ -139,3 +147,16 @@ def is_valid_ipfs_uri(ipfs_uri):
     except Exception as err:
         logger.exception('Unexpected error: %s', err)
     return False
+
+
+def run_in_executor(f):
+    """
+    Decorator. Runs the function it decorates in an executor.
+    """
+    def inner(*args, **kwargs):
```

Comment:
Is a [good pratice](https://docs.python.org/3.8/library/functools.html#functools.wraps) to not lose [the metadata](https://stackoverflow.com/a/55102697/798575) of the decorated func when decorating it:
```suggestion
    @functools.wrap(f)
    def inner(*args, **kwargs):
```
(if accepting the suggestion, please remember to `import functools` ;) )